sse-event-handler/target/classes/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,21 @@
+Manifest-Version: 1.0
```

Comment:
remove /target folder from the comments. We need to put only the source files. Let's remove the target files from committing. You need to add target to gitignore file too https://github.com/wso2-extensions/identity-outbound-auth-email-otp/blob/master/.gitignore#L2